### ISSUE TITLE
请教Offline评测数据集上的训练设置

### ISSUE THREAD

**Issue by XuRui314** [OS-Copilot/OS-Atlas#45]
论文里没有提到各个数据集上的微调设置，Table 4中提到了Qwen2VL-7B（baseline）在AndroidControl上的准确率，想请问这个微调设置是什么，epoch 学习率 学习率下降策略等，以及能否提供AndroidControl测试数据对应episode的编号，即论文提到的7708个测试样本，完整的数据应该是大于这个数量的

**Comment by xufangzhi** [comment]
Hello，我们更新了AndroidControl的测试episode id，在`OS-Atlas/eval/data/ac_idx.txt`

**Comment by XuRui314** [comment]
请问微调设置可以指导吗，目前baseline没复现出来

**Comment by LividWo** [comment]
@xufangzhi @numbmelon

**Comment by numbmelon** [comment]
The following hyperparameters were used during training:
- learning_rate: 1e-05
- train_batch_size: 1
- eval_batch_size: 8
- seed: 42
- distributed_type: multi-GPU
- num_devices: 8
- gradient_accumulation_steps: 16
- total_train_batch_size: 128
- total_eval_batch_size: 64
- optimizer: Adam with betas=(0.9,0.999) and epsilon=1e-08
- lr_scheduler_type: cosine
- lr_scheduler_warmup_ratio: 0.03
- num_epochs: 1.0

**Comment by XuRui314** [comment]
收到




Mr.Pluskid
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "OS-Copilot/OS-Atlas"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年2月20日(星期四) 下午3:24
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [OS-Copilot/OS-Atlas] 请教Offline评测数据集上的训练设置 (Issue #45)



   
The following hyperparameters were used during training:
  
learning_rate: 1e-05
 
train_batch_size: 1
 
eval_batch_size: 8
 
seed: 42
 
distributed_type: multi-GPU
 
num_devices: 8
 
gradient_accumulation_steps: 16
 
total_train_batch_size: 128
 
total_eval_batch_size: 64
 
optimizer: Adam with betas=(0.9,0.999) and epsilon=1e-08
 
lr_scheduler_type: cosine
 
lr_scheduler_warmup_ratio: 0.03
 
num_epochs: 1.0
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***&gt;
  numbmelon left a comment (OS-Copilot/OS-Atlas#45)
 
The following hyperparameters were used during training:
  
learning_rate: 1e-05
 
train_batch_size: 1
 
eval_batch_size: 8
 
seed: 42
 
distributed_type: multi-GPU
 
num_devices: 8
 
gradient_accumulation_steps: 16
 
total_train_batch_size: 128
 
total_eval_batch_size: 64
 
optimizer: Adam with betas=(0.9,0.999) and epsilon=1e-08
 
lr_scheduler_type: cosine
 
lr_scheduler_warmup_ratio: 0.03
 
num_epochs: 1.0

**Comment by lijianhong-code** [comment]
> 请问微调设置可以指导吗，目前baseline没复现出来

我目前也并没有复现处理，ac test是65%左右，差距还是存在
